claims_errors.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021-2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psatoken
+
+import "errors"
+
+var (
+	ErrClaimUndefined        = errors.New("undefined claim")
+	ErrOptionalClaimMissing  = errors.New("missing optional claim")
+	ErrMandatoryClaimMissing = errors.New("missing mandatory claim")
+	ErrWrongSyntax           = errors.New("wrong syntax for claim")
```

Comment:
I am little nervous about the usage of term: ErrWrongSyntax, which is more suitable for a language.

Can it be reworded as :
ErrWrongClaimValue or ErrWrongValue??

Reply:
I'd suggest using `ErrWrongClaimSyntax`

Reply:
ok for me!

Reply:
done in [505b6bb](https://github.com/veraison/psatoken/pull/26/commits/505b6bb1d63ca0036a950eaca3063f4b8cae053e)

Reply:
Thanks!